src/components/Logo/Logo.module.css

```
@@ -5,17 +5,17 @@
 \*------------------------------------*/
 
 /**
- * Branding image or text of the site
- * 1) Wrapper (usually a link) that contains the logo graphic
- * 2) Width always set to fit content within whether it's an SVG or an img
+ * Branding image or text of the site.
  */
 
 /**
  * Logo link 
  */
+ /* Wrapper (usually a link) that contains the logo graphic. */
```

You can just leave an empty line between comments in the same css comment block.
```suggestion
 *
 * Wrapper (usually a link) that contains the logo graphic.
 */
```